### ISSUE TITLE
Highlight color of `LspSigActiveParameter` is probably not appropriate.

### ISSUE THREAD

**Issue by beavailable** [yegappan/lsp#456]
I use [onedark](https://github.com/joshdick/onedark.vim) as the theme for vim, I found that the color of `LspSigActiveParameter` is gray that I can barely see it.

After some research, I found out that the color is linked to `LineNr`:
```vim
hlset([{name: 'LspSigActiveParameter', default: true, linksto: 'LineNr'}])
```

Many users use a dark theme for vim and the color of `LineNr` is gray, so I thought that linking `LspSigActiveParameter` to `LineNr` may not be a good idea.

Additionally, I think that linking `LspSigActiveParameter` to `Special` may be good (at least in my case) and the color is blue in `onedark` theme.

**Comment by yegappan** [comment]
You can override the highlight groups used by the plugin.  In your .vimrc file, before adding the plugin, you can define the highlight group:
```
   highlight link LspSigActiveParameter Special
```

In my local Ubuntu setup, I see that the LineNr highlight group is using the Yellow foreground color and on MS-Windows, it is using Brown.  Are you using a color scheme, that changes this highlight group?

**Comment by beavailable** [comment]
Yes, `onedark` changes the color of `LineNr` and I think that's good, because people don't need a bright color for line number, but they need a bright color for `LspSigActiveParameter`.
What I mean is that linking `LspSigActiveParameter` to `LineNr` is not appropriate for many users who're using a dark theme and not appropriate semantically.

I could change this in my `vimrc`, but I still think it's a better idea to link to `Special`. However, it's up to you.
